lib/index.js

```diff
@@ -38,7 +54,10 @@ export default class Browsochrones extends WebWorkerPromiseInterface {
     })
   }
 
-  createTile = (coords, done) => {
+  createTile = (
+    coords: Coords,
+    done: Function
```

Comment:
[The Function type is unsafe and should be avoided](https://flow.org/en/docs/types/functions/#toc-function-type).